include/maliput/api/maliput_design.h

```diff
@@ -160,8 +160,60 @@
 ///
 /// @subsubsection scale_length_and_tolerances Scale Length and Tolerances
 ///
-/// > TODO: Explain the concepts of linear tolerance, angular tolerance, and
-/// > characteristic scale length.
+/// The set of abstractions conformed by the `RoadGeometry`, `Junction`, `Segment`,
+/// `Lane` and `BranchPoint` concentrate various aspects of map data, among which we can
+/// identify and highlight:
+/// - Network topology.
+/// - Volume and surface information.
+/// - Dimensions.
+///
+/// Depending on the data source type that will feed an actual implementation of `maliput`
+/// these three aspects will be given, derived or found at the moment of parsing the information
+/// and building the entities. In particular, there is one important characteristic of the
+/// `RoadGeometry` abstraction: the @f$ G^1 @f$ continuity of the road surface. That implies:
+/// - The road surface is continuous.
+/// - The road surface is orientable.
+///
+/// Provided the available map information is imperfect, the numeric system under use
+/// is also imperfect and discrete and there are non-continuities that are by far neglactable
+/// the use of tolerances becomes relevant for designers. The `maliput` API offers
```

Comment:
nit: Fix semantics between "...are by far neglactable" and "...the use of tolerances becomes relevant for designers". Missing a connector

Reply:
I have written the paragraph again. PTAL in 0318583